components/UI/Sidebar.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
Don't create a new sidebar component.
Please reuse the one we already developped components/domains/steps/registerSteps.tsx.
You'll just need to add new props to make it reusable